docs/edgeql/statements/insert.rst

```diff
@@ -46,6 +49,23 @@ See :ref:`Usage of FOR statement <ref_eql_forstatement>` for more details.
         INSERT <expression>
         [ "{" <link> := <insert-value-expr> [, ...]  "}" ]
 
+:eql:synopsis:`UNLESS CONFLICT ON <property>`
+    Handler of conflicts.
+
+    This clause allows to handle specific conflicts arising during
+    execution of ``INSERT`` without producing an error.  If the
+    conflict arises due to constraints on the specified *property*,
+    then instead of failing with an error the ``INSERT`` statement
+    produces an empty set (or an alternative result).
+
+:eql:synopsis:`ELSE <alternative>`
+    Alternative result in case of conflict.
+
+    This clause can only appear after ``UNLESS CONFLICT`` clause. Any
+    valid expression can be specified as the *alternative*. When a
+    conflict arises, the result of the ``INSERT`` becomes the
+    *alternative* expression (instead of the default ``{}``).
```

Comment:
Document the binding rules: in `INSERT Foo ... UNLESS CONFLICT ELSE`, `Foo` is bound to the conflicting object in the `ELSE` clause.

Reply:
OK, updated and added a link to the actual example that appears later in the section.